src/mokernels/moinput.jl

```diff
@@ -194,3 +194,5 @@ function prepare_isotopic_multi_output_data(x::AbstractVector, y::RowVecs)
     length(x) == length(y) || throw(ArgumentError("length(x) not equal to length(y)."))
     return MOInputIsotopicByOutputs(x, size(y.X, 2)), vec(y.X)
 end
+
+MOInputsUnion = Union{MOInputIsotopicByFeatures,MOInputIsotopicByOutputs}
```

Comment:
```suggestion
MOInputsIsotopic = Union{MOInputIsotopicByFeatures,MOInputIsotopicByOutputs}
```
or
```suggestion
IsotopicMOInputs = Union{MOInputIsotopicByFeatures,MOInputIsotopicByOutputs}
```
- happy to keep the `Union` suffix as well - but it's crucial that this makes clear it's about *isotopic* MO inputs